### ISSUE TITLE
Questions about ID classification and running on other datasets

### ISSUE THREAD

**Issue by linhaowei1** [deeplearning-wisc/cider#3]
Hi authors,

Thanks for your fantastic work! I am following this work and use this repo these days. The code is of high quality, too.
BTW, here is a minor issue in the code:
- The argument ``--w`` in ``train_cider.py`` is inconsistent with `--w_dis` in the training scripts.

I have two questions about cider:
- How do you perform the ID classification (the specific training epochs, learning rate)? I refer to relevant papers such as SimCLR and CSI and implemented by myself, but the final results are worse than those reported in the paper. 
- Can you give some advice on training on datasets with fewer classes? I trained CIDER on 2 classes of CIFAR-10, but the two losses fail to balance -- using the original hyper-parameters, the projection head will collapse to a constant mapping (with the largest `disLoss` and lowest `compLoss`). I observe this issue happens when the number of ID classes are small, and using 8 classes of CIFAR-10 is fine.

**Comment by alvinmingwisc** [comment]
Sorry for the delay and thanks very much for your interest and pointing out the mismatch! Just fixed. 

The ID performance is typically evaluated with linear probe (i.e., train a linear classifier on top of pre-trained feature representation). In our experience, the hyperparameters do not much affect the ID accuracy. Here we used the scripts in the SupCon repo for ID evaluation: https://github.com/HobbitLong/SupContrast 

Please let us know if you have further questions.
